File: 1010.c
#include <stdio.h>
int main(){
    int qtde1, qtde2, cod1, cod2;
    double valor1, valor2, total;
    scanf("%i%i%lf", &cod1, &qtde1, &valor1);
    scanf("%i%i%lf", &cod2, &qtde2, &valor2);
    total = (qtde1 * valor1) + (qtde2 * valor2);
    printf("VALOR A PAGAR: R$ %.2lf\n",total);
    printf("");
}

File: 1010.java
import java.util.Scanner;
import java.util.Locale;
public class Main {
public static void main(String[] args){
        Locale.setDefault(Locale.US);
        Scanner input = new Scanner(System.in);
        int cod1 = input.nextInt(); int qtde1 = input.nextInt(); double valor1 = input.nextDouble();
        int cod2 = input.nextInt(); int qtde2 = input.nextInt(); double valor2 = input.nextDouble();
        double total = (qtde1 * valor1) + (qtde2 * valor2);
        System.out.printf("VALOR A PAGAR: R$ %.2f%n", total);
        }
}
